Make-a-pull-request.md

```diff
@@ -0,0 +1,168 @@
+
+4. **When your feature is ready to merge, create a pull request.**
+
+   * Go to your fork on GitHub, select your branch from the dropdown menu, and click "pull request". Ensure that the base repository is `oppia/oppia` and that the base branch is `develop`. The head repository should be your fork, and the head branch should be your branch. If you don't see the repository, click the link to `compare across forks`.
+
+     On this page you can also see your commits and your changes.  Read these changes carefully to make sure that the changes are correct (e.g., no missing newlines at the ends of files; no files left out by mistake). This is a good way to catch obvious errors that would otherwise lead to delays in the review process.
+
+     ![The GitHub PR creation screen](images/makingAPR.png)
+
+   * Click "Create pull request".
+
+   * Following the guidance in the PR checklist, add a descriptive title explaining the purpose of the PR (e.g. "Fix issue #bugnum: add a warning when the user leaves a page in the middle of an exploration.").
+
+   * Fill out the rest of the PR checklist.
+
+   * **Test your code to confirm that the changes you have made are working properly** Test all the places where the changes are made, including buttons, tabs, lists etc. Make a screen recording of these changes and add it to the description.
+
+   * Click "Create pull request".
+
+   * If you have already completed 2 pull requests and been added as a collaborator to the project, you should also add a changelog label.  If you are a new contributor, you don't have permission to do this.  Don't worry! Oppiabot will automatically ask someone to do it for you.
+
+   * Oppiabot will check that you filled out the PR description correctly. If you didn't, it will leave a comment explaining what you need to do to fix it.
+
+   * GitHub will automatically assign reviewers, and Oppiabot will assign the issue's "owner" (from the changelog label) to do the first review.
+
+     If you need to assign someone else but aren't a collaborator yet, leave a comment of the form `@{{reviewer username}} PTAL`, which will tell Oppiabot to assign that person for you. ("PTAL" means "Please take a look".)
+
+   * After a while, check your pull request to see whether the CI checks have passed. If not, follow the instructions at [[If your build fails|If-your-build-fails]].
+
+   * Then, wait for your code to get reviewed! While you're waiting, it's totally fine to start work on a new PR if you like. Just make sure to **checkout the develop branch** and pull in changes from upstream before you check out a new branch, so that each of your feature branches is based off of `develop`.
```

Comment:
I think we should leave the concept of working on multiple branches simultaneously until the end. At the beginning, that feels like unnecessary info that might overwhelm readers who are just getting started in open source

Reply:
Ah, sorry for the misunderstanding, I was suggesting to change the wordings here and ask them to follow this instruction from the beginning if they are planning to work on a new issue in parallel. (Instead of writing steps like `checkout the develop branch` etc.. over here!)

Reply:
@DubeySandeep I don't think I understand. I'm hearing that you want to move this suggestion to the beginning of the docs like this:

> (At the start of these docs): If you want to work on multiple issues in parallel, make sure you create a branch for each issue off of the latest version of `develop`
>
> ... (The docs that come before this comment) ...
>
> * Then, wait for your code to get reviewed! While you're waiting, it's totally fine to start work on another PR.

Or, do you mean that we should say something like this:

> * Then, wait for your code to get reviewed! While you're waiting, it's totally fine to start work on another PR. Just follow these instructions again for that PR.

Reply:
Yeah something around the latter one, i.e, follow the instructions from step 1. (for creating your next PR)

Reply:
done